wordpress-mobile/AztecEditor-iOS
# Issue/605 build step check for carthage update

Fixes #
build step to run carthage update command
To test:
download the source code from repo
run/build/test/profile/archive the project
check whether a terminal window opens up, and execute the 'carthage update' based on the existence of the folder 'iOS' inside Carthage > Build
